tests/test_helpers/test_test_path_scope.py

```
@@ -0,0 +1,78 @@
+import os
+import subprocess
+from pathlib import Path
+
+REPO_ROOT = Path(__file__).resolve().parents[2]
+SCOPE_SH = REPO_ROOT / "scripts" / "test_path_scope.sh"
+
+
+def _filter(test_path: str, files: str) -> str:
+    env = os.environ.copy()
+    env["TEST_PATH"] = test_path
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)